tests/integration_tests/allocation_entries/test_get.py

```
@@ -0,0 +1,54 @@
+from tests.integration_tests.base.rest_client import RestClient
+from tests.integration_tests.base.soap_client import SoapClient
+
+
+@pytest.mark.integration
+class TestAllocationEntriesGet(BaseTestGet):
+    """
+    Test allocation entries GET operations.
+    """
+    REST_MODULE_NAME = 'allocations'
+    SOAP_MODULE_NAME = 'allocation_entry'
+    ALLOWED_METHODS = ['get_all']
```

Ruff (0.14.3)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)